CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## [v1.6.0](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.6.0) (2022-11-16)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.5.0...v1.6.0)
+
+**NEW FEATURES:** 
```

Comment:
Not blocking but last time we used ENHANCEMENTS and now it's NEW FEATURES - we need a defined format and a script to generate so we have consistency in the CHANGELOG so it's easy to read.